pubspec.yaml

```diff
@@ -31,14 +31,15 @@ dependencies:
   freezed_annotation: ^2.0.3
   go_router: ^9.0.0
   intl: ^0.18.0
-  json_annotation: ^4.5.0
+  json_annotation: ^4.8.1
   package_info_plus: ^3.0.0
   permission_handler: ^10.3.0
   retrofit: ^4.0.1
   injectable: ^1.5.3
   get_it: ^7.2.0
   flutter_riverpod: ^1.0.3
   flutter_svg: ^1.0.3
+  shared_preferences: ^2.1.4
```

Comment:
I recommend utilizing [flutter_secure_storage](https://pub.dev/packages/flutter_secure_storage) to store credential values instead of using `shared_preferences`. Flutter Secure Storage provides a more `secure` and `encrypted` way to store sensitive data, ensuring better protection of credentials. 🙏

Reply:
@Wadeewee resolved in https://github.com/nimblehq/ic-flutter-avishek/pull/34/commits/9a8bd23566f92cf478aba25df9d9d71f41d70585